tests/test_fileIO.py

```diff
@@ -27,4 +28,27 @@ def test_read_dynamx(self):
         assert data['end'][0] == 17
 
         with pytest.raises(ValueError):
-            data = read_dynamx(self.fpath, intervals=('foo', 'bar'))
+            data = read_dynamx(self.fpath, intervals=('foo', 'inclusive'))
+        with pytest.raises(ValueError):
+            data = read_dynamx(self.fpath, intervals=('inclusive', 'bar'))
+
+        with open(self.fpath, mode='r') as f:
+            read_dynamx(StringIO(f.read()))
+
+
+    def test_methods(self):
+        data = read_dynamx(self.fpath)
+        path = directory / 'test_data' / 'ecSecB_guess.txt';
+
+        #testing fmt_export
+        with pytest.raises(ValueError):
+            fmt_export(data, delimiter=',', width="foo")
+
+        #testing txt_to_protein
+        txt_to_protein(path)
```

Comment:
This is good but you should include an `assert` when you add a function for testing. Here the `txt_to_protein` returns an object, so you could write an assert to check if this is the correct object, and then another which checks if the data in the object is as expected